packages/three-vrm-materials-mtoon/src/shaders/mtoon.frag

```diff
@@ -580,16 +615,29 @@ void main() {
   // Since we want to take shadows into account of shading instead of irradiance,
   // we had to modify the codes that multiplies the results of shadowmap into color of direct lights.
 
-  GeometricContext geometry;
+  #if THREE_VRM_THREE_REVISION >= 157
+    vec3 geometryPosition = - vViewPosition;
+    vec3 geometryViewDir = ( isOrthographic ) ? vec3( 0, 0, 1 ) : normalize( vViewPosition );
+    
+    vec3 geometryClearcoatNormal;
+
+    #ifdef CLEARCOAT
+
+      geometryClearcoatNormal = clearcoatNormal;
+
+    #endif
+  #else
+    GeometricContext geometry;
 
-  geometry.position = - vViewPosition;
-  geometry.normal = normal;
-  geometry.viewDir = ( isOrthographic ) ? vec3( 0, 0, 1 ) : normalize( vViewPosition );
+    geometry.position = - vViewPosition;
+    geometry.normal = normal;
+    geometry.viewDir = ( isOrthographic ) ? vec3( 0, 0, 1 ) : normalize( vViewPosition );
 
-  #ifdef CLEARCOAT
+    #ifdef CLEARCOAT
```

Comment:
This line also. The support of r131 is already dropped so we can safely rename this

```suggestion
    #ifdef USE_CLEARCOAT
```